### DISCUSSION TITLE
Ft Req / Inquiry : Github app for third party

### DISCUSSION THREAD

**Discussion by plowsof** [coollabsio/coolify#3081] [Feature Requests]
Hello, maybe this is possible already but i have struggled with this. I would like to provide preview deployments (comments via the github app) on a public repo from a third party. My thought process was for them to install a github app, and then i would manually go into coolify and replace the details (product id ,  secret and such) but those  fields appear to not be editable 'after the fact'.

 The goal being that the third party does not have to log in to my coolify instance and set things up (i do not want to en-danger them or the repo they control due to my own lack of security). I would much rather them install an app without touching my coolify instance then hand me the details i input for access. Is this possible already? 

Thanks for reading

update: perhaps this is what im looking for https://github.com/marketplace/actions/coolify-application-deploy

update 2: i am currently using webhooks. the third party repo maintainer just adds the webhook url / secret and notifies on pullrequest/push , but i lose the coolify app which comments

**Comment by plowsof** [comment]
bump (: using webhooks for deployment with no app on the repo - would be great if i could manually add APP keys to coolify

**Comment by nahuhh** [comment]
https://github.com/monero-project/monero-docs/blob/master/.github/workflows/comment_pr.yml

We were able to use a workflow to comment the preview deployment links w/o using the coolify github app
